public_dropin_gpu_environments/python311_genai/env_info.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "64d2ba178dd3f0b1fa2162f0",
+  "name": "[DataRobot] Python 3.11 GenAI",
```

Comment:
Please change the name and description, otherwise it makes it the same as existing one.

Reply:
Is there best practise how to mark that this env is for GPU in the name ?

Reply:
Probably yes. Just to make it clear

Reply:
Make it clear in the name and details in description.